python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml

```diff
@@ -276,11 +308,51 @@ keys:
                       description: Client Inner VLAN ID.
                 unmatched:
                   type: bool
+                  deprecation:
+                    warning: true
+                    remove_in_version: 6.0.0
+                # TODO: AVD6.0 the `protocol` key should be made required once the deprecated keys are removed in AVD6.0
+                protocol:
+                  type: str
+                  valid_values:
+                    - dot1q
+                    - dot1ad
+                    - unmatched
+                    - untagged
+                vlan:
+                  type: int
+                  convert_types:
+                    - str
+                  min: 1
+                  max: 4094
+                  description: Client VLAN ID. Not applicable for `protocol==untagged/unmatched`.
+                outer:
```

Comment:
may be we can make it outer_vlan since it is vlan id.

similarly, inner_vlan below as well.

Reply:
updated!